### ISSUE TITLE
Earlier messages flash over then latest messages show up

### ISSUE THREAD

**Issue by TaoistKing** [jessesquires/JSQMessagesViewController#1924]
## New issue checklist

- [ YES] I have read all of the [`README`]
- [ YES] I have reviewed the [contributing guidelines]
- [ YES] I have searched [existing issues]

## General information

- Library version(s): 7.3.4
- iOS version(s): 10.0.2
- Devices/Simulators affected: Device
- Reproducible in the demo project? (Yes/No): No
- Related issues:

## What happened?
Hi , There is noticeable view transform after JSQ is loaded in my App. 
But seems Not reproducible in the Demo.

I add some log and found out, JSQ will ask twice for messages.
I have 100 messages in the datasource, JSQ will ask for message 0 ~ 10 at the first time, 
then asks for message 90 ~ 100 for the second time, 
So when view appeared, the first 0 ~ 10 messages will flash over and then message 90 ~ 100 shows up(the latest messages).

I'm new to JSQ, please help me out, thanks.

**Comment by TaoistKing** [comment]
Seems It's only asking for messages once, numberOfItemsInSection is called only once.
but It will ask for 0~ 10 and then 90 ~ 100.
I guess It's asking for messages in front of datasource upon loading , then JSQ automaticallyScrollsToMostRecentMessage , then it's asking messages at the end, so the view jumps from beginning to end.

**Comment by TaoistKing** [comment]
ghazel's answer in [780](https://github.com/jessesquires/JSQMessagesViewController/issues/780) fixed this.

**Comment by jessesquires** [comment]
Thanks @TaoistKing ! 👍
